src/Projects/IrcClient/Commands/Helpers/DccCommandHelper.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace IrcClient.Commands.Helpers
+{
+    internal static class DccCommandHelper
+    {
+        private static readonly Dictionary<DccCommand, string> DccCommandString = new Dictionary<DccCommand, string>()
+        {
+            { DccCommand.Accept, "ACCEPT" },
+            { DccCommand.Chat, "CHAT" },
+            { DccCommand.Recv, "RECV" },
+            { DccCommand.Reject, "REJECT" },
+            { DccCommand.Resume, "RESUME" },
+            { DccCommand.Reverse, "REVERSE" },
+            { DccCommand.RSend, "RSEND" },
+            { DccCommand.Send, "SEND" },
+            { DccCommand.Xmit, "XMIT" }
```

Comment:
I feel like it would be cleaner to remove that dictionary and do the "translation" string<>enum by name convention.
Like:
`return command.ToString().ToUpper();` for ToCommandString method.

Reply:
As said in the description, this pr is solely about the horriblesubs module the code for the irc client will be reviewed and rewritten separately